docs/vcs.rst

```diff
@@ -228,52 +220,50 @@ GitHub
 
 .. versionadded:: 2.3
 
-This just adds a thin layer on top of :ref:`vcs-git` to allow push translation
-changes as pull requests instead of pushing directory to the repository.
-It currently uses the `hub`_ tool to do the integration.
+This adds a thin layer atop :ref:`vcs-git` using the `hub`_ tool to allow pushing
+translation changes as pull requests, instead of
+pushing the directory to the repository.
 
-There is no need to use this to access Git repositories, ordinary
-:ref:`vcs-git` works the same, the only difference is how pushing to a repository is
-handled. With :ref:`vcs-git` changes are pushed directly to the repository, while
+:ref:`vcs-git` pushes changes directly to a repository, while
 :ref:`vcs-github` creates pull requests.
+The latter is not needed for merely accessing Git repositories.
 
 .. _github-push:
 
-Pushing changes to GitHub as pull request
+Pushing changes to GitHub as pull requests
 +++++++++++++++++++++++++++++++++++++++++
 
-If you are translating a project that's hosted on GitHub and don't want to
-push translations to the repository, you can have them sent as a pull request instead.
+If not wanting to push translations to a GitHub repository, they can be sent as either
+one or many pull requests instead.
 
-You need to configure the `hub`_ command line tool and set
-:setting:`GITHUB_USERNAME` for this to work.
+Configure the `hub`_ command line tool and set :setting:`GITHUB_USERNAME` for this to work.
 
 .. seealso::
 
    :setting:`GITHUB_USERNAME`, :ref:`hub-setup` for configuration instructions
 
 .. _hub-setup:
 
-Setting up hub
+Setting up Hub
 ++++++++++++++
 
 :ref:`github-push` requires a configured `hub`_ installation on your server.
-Follow the installation instructions at https://hub.github.com/ and perform an
-action with `hub`_ to finish the configuration, for example:
+Follow the installation instructions at https://hub.github.com/ use `hub`_ to
+finish the configuration, for example:
 
 .. code-block:: sh
 
-    # DATA_DIR is set in Weblate settings.py, set it accordingy.
+    # DATA_DIR is configured in Weblate settings.py, set it accordingy.
```

Comment:
According to?

Reply:
To match what is specified in the settings.py